invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -66,14 +66,28 @@
   min-width: 10em;
 }
 
-.ui.dropdown.fluid-mobile {
-  @media all and (max-width: @largestTabletScreen) {
-    width: 100%;
-    min-width: 0em;
-    display: block;
-
-    & > .dropdown.icon {
-      float: right;
+.ui.dropdown {
+  &.fluid-tablet{
+    @media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen) {
+      width: 100%;
+      min-width: 0em;
```

Comment:
I am not convinced for the min-width:0, I think we should have there a better value adjusted per device, otherwise all the dropdowns will have random minimal lengths based on their content